### DISCUSSION TITLE
Grab Amount?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#402] [Q&A] [answered]
Basically /grab [item] [amount]

**Comment by winstxnhdw** [comment]
What’s the use for this? Just curious.

**Reply by Dexteriority** [reply]
So we can adjust the items in the ship xD
So I grab V-Engine about 12 of them and had it spread around, so it's difficult with F and G, maybe with this command we can bring a certain amount of item in a position we are in.


ITS COMPLICATED BUT ACTUALLY USELESS? 😭😭😭😭😭

**Reply by winstxnhdw** [reply]
I am lost.

**Reply by Dexteriority** [reply]
i know, im dumb at explaining but the other purpose is for us to grab whats only needed from the facility not all :p

**Reply by winstxnhdw** [reply, marked as answer]
Sure, but isn’t the `grab <item>` command sufficient?

**Reply by Dexteriority** [reply]
It is though!!
I'll stay on using **/grab [item]**.
Thanks for the reply!!!

**Reply by winstxnhdw** [reply]
If there’s a valid reason I don’t realise, just let me know.
